### DISCUSSION TITLE
Adding label to Filter Text Fields

### DISCUSSION THREAD

**Discussion by BennyAlex** [KevinVandy/material-react-table#479] [General]
Is there a way to move the placeholder text to the input label instead?
![image](https://user-images.githubusercontent.com/17402724/232862306-3d1efc68-4759-4bec-a3fb-94389a2225e2.png)

The table content itself is not wide, but the min & max filtes together are, so the placeholder text is not visible.
If it would be a label on the top, it would be readable.

**Comment by KevinVandy** [comment]
Sure, with your own muiTableHeadCellFilterTextFieldProps, you can customize anything

**Reply by BennyAlex** [reply]
Okay can you explain how, please?
I cant find the placeholder text in (table, columns).
Is the function called for both text fields seperatly?

**Reply by KevinVandy** [reply]
I'm assuming you have seen the external link to the MUI docs for TextField props? https://mui.com/material-ui/api/text-field/

**Reply by BennyAlex** [reply]
Yeah sure, I know I can add a Label to it, but when the table is calling the muiTableHeadCellFilterTextFieldProps function, I get an object with table and columns.
But I dont know where exactly the text which normally is written in the placeholder is generated or how to access it, so I can write it into the label instead.

**Reply by KevinVandy** [reply]
muiTableHeadCellFilterTextFieldProps: {
  label: 'Your label',
}

**Reply by BennyAlex** [reply]
yeah sure thanks for trying to help. But I dont know what type of field this is. For example, the range filter has an min and max field. So I somehow need to know what textfield for what filter this is.
